boards/shields/st_b_lcd40_dsi1_mb1166/Kconfig.defconfig

```diff
@@ -8,19 +8,13 @@ if SHIELD_ST_B_LCD40_DSI1_MB1166 || SHIELD_ST_B_LCD40_DSI1_MB1166_A09
 if LVGL
 
 config STM32_LTDC_FB_NUM
-	default 0
+	default 1
 
 config INPUT
 	default y
 
 config LV_Z_VDB_SIZE
-	default 100
-
-config LV_Z_DOUBLE_VDB
-	default y
-
-config LV_Z_VBD_CUSTOM_SECTION
-	default y
+	default 10
```

Comment:
`LV_Z_VDB_SIZE` default value is already `10` in `modules/lvgl/Kconfig.memory`.

Reply:
when LV_Z_FULL_REFRESH  is selected then it turns out 100